tests/storages_tests/test_storages.py

```diff
@@ -472,6 +488,14 @@ def test_set_trial_state(storage_mode: str) -> None:
             else:
                 assert storage.get_trial(trial_id).datetime_complete is None
 
+        # Non-existent study.
+        with pytest.raises(KeyError):
```

Comment:
How about adding this type of test to the `test_set_trial_state_values_for_state`?

Reply:
You got the point. I added the same test to `test_set_trial_state_values_for_state` in [ab89ad8](https://github.com/optuna/optuna/pull/3480/commits/ab89ad8d79d497759f9268b13067a62e367113f4).